lib/checks/mobile/target-size-evaluate.js

```diff
@@ -10,7 +10,7 @@ import {
  * Determine if an element has a minimum size, taking into account
  * any elements that may obscure it.
  */
-export default function targetSize(node, options, vNode) {
+export default function targetSizeEvaluate(node, options, vNode) {
```

Comment:
We talked about adding an early pass for elements that have 10x the required size. I don't mind that in a different PR, but I don't think we should close the issues until we have that in.

Reply:
I think I'd prefer to move that to it's own pr so we can get this in quicker. Which ticket do you think it shouldn't close?